pkg/graph/resolvers/itgovactions/lcidactions/confirm_lcid.go

```diff
@@ -0,0 +1,39 @@
+package lcidactions
+
+import (
+	"time"
+
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// GetConfirmLCIDAction is a method to set all the relevant data fields for that are a result of an Update LCID action.
+func GetConfirmLCIDAction(
+	intake models.SystemIntake,
+	expirationDate time.Time,
+	nextSteps models.HTML,
+	scope models.HTML,
+	costBaseline *string,
+	userInfo models.UserInfo,
+) models.Action {
+
+	action := getBaseLCIDAction(intake, &expirationDate, &nextSteps, &scope, costBaseline, userInfo)
+	action.ActionType = models.ActionTypeCONFIRMLCID
+
+	return action
+
+}
+
+// IsLCIDValidToConfirm checks if you can confirm the LCID for an intake, if not, it will return an error
+// An LCID must have already been issued in order to be confirmable
+func IsLCIDValidToConfirm(intake *models.SystemIntake) error {
+	if !intake.LifecycleID.Valid {
```

Comment:
A `null.String` constructed from an empty string still has `.Valid` set to true (see https://goplay.tools/snippet/4smKC8zCV5P), so I'm not sure that this test is sufficient. What I've been doing for these situations is checking `intake.LifecycleID.ValueOrZero() == ""`, which returns true for both null strings and empty strings.